package.json

```diff
@@ -47,11 +47,11 @@
     "@typescript-eslint/parser": "^8.14.0",
     "angular-eslint": "18.4.0",
     "autoprefixer": "^10.4.20",
-    "eslint": "^9.14.0",
+    "eslint": "9.14.0",
```

Comment:
Pinned the version because of [this issue](https://github.com/typescript-eslint/typescript-eslint/issues/10338).

Reply:
Outdated, issue is solved now.